jest.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  projects: [
```

Comment:
[F] Running from the root will generate warnings but these are expected: https://github.com/facebook/jest/issues/13576